spec/stripe/remote/integration_spec.rb

```diff
@@ -190,4 +188,54 @@
     payment_response.status.should eq(:PROCESSED), payment_response.gateway_error
     payment_response.transaction_type.should == :VOID
   end
+
+  it 'should be able to add a bank account, verify and accept a bank charge' do
+    kb_payment_method_id = SecureRandom.uuid
+    pm = @plugin.add_payment_method(@pm.kb_account_id, kb_payment_method_id, bank_account_properties, true, [], @plugin.kb_apis.create_context(@call_context.tenant_id))
+    pm.token.should be_kind_of(String)
+    pm.stripe_customer_id.should be_kind_of(String)
+    pm.bank_name.should eq("STRIPE TEST BANK")
+    pm.bank_routing_number.should eq("110000000")
+    pm.country.should eq("US")
+    pm.source_type.should eq("bank_account")
+
+    properties = [build_property(:token, pm.token)]
+    kb_payment = @plugin.kb_apis.proxied_services[:payment_api].add_payment(SecureRandom.uuid)
+
+    attempt_response = @plugin.purchase_payment(@pm.kb_account_id, kb_payment.id, kb_payment.transactions[0].id, pm.kb_payment_method_id, @amount, @currency, properties, @call_context)
+    attempt_response.status.should eq(:ERROR), attempt_response.gateway_error
+
+    @plugin.verify_bank_account(pm.stripe_customer_id, pm.token, bank_account_verification_numbers, @call_context.tenant_id)
```

Comment:
Is there a way to check the verification was successful? The test is failing for me (wondering if there's a delay?):

```
Failures:

  1) Killbill::Stripe::PaymentPlugin should be able to add a bank account, verify and accept a bank charge
     Failure/Error: payment_response.status.should eq(:PROCESSED), payment_response.gateway_error
       The customer's bank account must be verified in order to create an ACH payment. For more information, see https://stripe.com/docs/guides/ach#manually-collecting-and-verifying-bank-accounts
       Diff:
       @@ -1,2 +1,2 @@
       -:PROCESSED
       +:ERROR

     # ./spec/stripe/remote/integration_spec.rb:211:in `(root)'
```

Reply:
That test ended up failing for me as well but I couldn't figure out a good reason for the failure so I thought I'd open the PR and see what happened for another developer.

I have been using the [Stripe Ruby client](https://github.com/stripe/stripe-ruby) to check the verification. If you use the same API key and query the bank account using it's token after `verify_bank_account` here the Stripe API returns that the card is verified. There is a `status` property for the card that is set to `verified` only after verification. So I'm not sure what the issue is, the error returned from Stripe does not fit with the other data being returned from their API calls. Maybe it is a delay? Maybe I should reach out to Stripe?

Reply:
I would indeed reach out to Stripe to understand if there is a delay.